docs/docsite/rst/network/getting_started/network_roles.rst

```diff
@@ -0,0 +1,307 @@
+
+.. _using_network_roles:
+
+*************************
+Use Ansible network roles
+*************************
+
+Roles are sets of Ansible defaults, files, tasks, templates, variables, and other Ansible components that work together. As you saw on :ref:`first_network_playbook`, moving from a command to a playbook makes it easy to run multiple tasks and repeat the same tasks in the same order. Moving from a playbook to a role makes it even easier to reuse and share your ordered tasks. You can look at :ref:`Ansible Galaxy <ansible_galaxy>`, which lets you share your roles and use others' roles, either directly or as inspiration.
```

Comment:
+ Playbooks?

Reply:
not sure what this comment is referring to?  Hard to  tell as it's an entire paragraph. If it's something simple, do you want to just use 'suggested edits' button above so I can see directly what you'd like changed?